folly/portability/Windows.h

```diff
@@ -37,6 +37,7 @@
 #include <direct.h> // nolint
 #endif
 
+#define NOMINMAX
```

Comment:
You need quite a few other defines to prevent a lot of other warnings being generated if you want to use more than a select few parts of Folly. See https://github.com/facebook/folly/blob/master/CMake/FollyCompiler.cmake#L226-L232

In theory, some of them can be shifted into here, but some extra work will be required to shift the rest, along with more beyond that to deal with the defines to keep it from warning on all of the use of the non-underscore-prefixed version of a lot of functions.